lib/rpcServer/commands/platform/getDataContractHandlerFactory.js

```diff
@@ -0,0 +1,60 @@
+const {
+  server: {
+    error: {
+      InvalidArgumentGrpcError,
+    },
+  },
+} = require('@dashevo/grpc-common');
+
+const RPCError = require('../../../rpcServer/RPCError');
+const Validator = require('../../../utils/Validator');
+const argsSchema = require('./schemas/getDataContract');
+const ValidationError = require('../../../errors/ArgumentsValidationError');
+
+const validator = new Validator(argsSchema);
+
+/**
+ *
+ * @param {DriveAdapter} driveAPI
+ * @param {DashPlatformProtocol} dpp
+ * @returns {getDataContractHandler}
+ */
+function getDataContractHandlerFactory(driveAPI, dpp) {
```

Comment:
I think it's better to pass the validator to the factory

Reply:
Fixed